src/components/AboutDescription/style.module.scss

```diff
@@ -2,34 +2,67 @@
   color: #f0f0f0;
   width: 95%;
   margin-top: 25px;
-  margin-left: 45px;
+  margin-left: 60px;
 
   .header {
     font-family: "Rubik", sans-serif;
     font-weight: 600;
-    font-size: 77px;
+    font-size: 70px;
     color: #ff8181;
-    line-height: 85px;
+    line-height: 90px;
     letter-spacing: 2px;
   }
 
   .descriptionBlock {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    column-gap: 20px;
+    column-gap: 80px;
 
-    span {
+    > span {
       font-weight: 400;
       font-family: "Lato", sans-serif;
       font-size: 33px;
       line-height: 40px;
       color: #ffffff;
+      margin-top: 20px;
     }
 
     .img {
       width: 400px;
       margin-right: 120px;
     }
   }
+  @media only screen and (max-width: 1100px) {
```

Comment:
A good convention for responsive media queries is to use standard breakpoint values unless you really can't avoid it.

![Screen Shot 2022-10-11 at 6 56 26 PM](https://user-images.githubusercontent.com/33165426/195231045-8f1a2d36-a30e-4c3e-9a62-cc6f0067c1d4.jpg)

1024px for desktop

![Screen Shot 2022-10-11 at 6 56 56 PM](https://user-images.githubusercontent.com/33165426/195231099-1a841e0a-ef8c-440a-94d0-85373cc172a6.jpg)

768px for tablet views

and 425px for mobile devices.

Basically if you need to write media queries with width breakpoints ever, try to handle it at one of these values. 425px for mobile is also usually the smallest viewport we care about so I would test the UI responsiveness up to that point